modules/ocf/manifests/conf.pp

```diff
@@ -0,0 +1,21 @@
+define ocf::conf(
```

Comment:
I would name this something more descriptive, perhaps "ocf::configbuilder" or something like that?